daemon/cmd/daemon_main.go

```diff
@@ -1590,6 +1593,7 @@ func initKubeProxyReplacementOptions() {
 		option.Config.EnableHostServicesUDP = true
 		option.Config.EnableSessionAffinity = true
 		option.Config.DisableK8sServices = false
+		option.Config.EnableHealthCheckNodePort = true
```

Comment:
In the same function, if we detect that `!option.Config.EnableNodePort` (line 1779), we should disable `option.Config.EnableHealthCheckNodePort` too.

Reply:
Done.